chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ActivityController.java

```diff
@@ -565,7 +580,9 @@
     }
 
     @ApiOperation(value = "Get auditable data about all CHPL user accounts",
-            notes = "Users must specify 'start' and 'end' parameters to restrict the date range of the results.")
+            notes = "Users must specify 'start' and 'end' parameters to restrict the date range of the results.  "
+                    + "Security Restrictions: ROLE_ADMIN, ROLE_ONC_ADMIN, ROLE_ONC_STAFF (of ROLE_ONC_STAFF), ROLE_CMS_STAFF "
```

Comment:
ROLE_ONC_ADMIN never existed, and ROLE_ONC_STAFF was rolled into ROLE_ONC

Reply:
ROLE_ONC_ADMIN was in the spreadsheet.  Went though and replaced all with ROLE_ONC.